kadi/commands/tests/test_commands.py

```diff
@@ -1154,7 +1154,6 @@ def test_get_cmds_from_event_case(par_str):
     Load not run,OCT2521A
     Command,"ACISPKT | TLMSID= AA00000000, CMDS= 3, WORDS= 3, PACKET(40)= D80000300030603001300"
     Command not run,"COMMAND_SW | TLMSID=4OHETGIN, HEX= 8050300, MSID= 4OHETGIN"
-    RTS,"RTSLOAD,1_4_CTI,NUM_HOURS=39:00:00,SCS_NUM=135"
```

Comment:
It wasn't clear to me if this RTS test from the event is needed but I moved it into its own (short) list to make it easier to conditionally skip.